client/lib/user-agent-utils/index.js

```diff
@@ -0,0 +1,7 @@
+export function currentUserAgent() {
+	return navigator.userAgent;
+}
```

Comment:
> we don't gain much right now as far as testability and server-side rendering is concerned by putting this behind a function, but there might be a way to provide that benefit

If we wanted to make this module SSR-compatible, we'd need to guess the UA on the server side from the HTTP headers — which isn't as reliable as it is on the client. I don't think we need this currently.

I'm fine with leaving this as is, seeing as the [`PragmaChecker`](https://github.com/Automattic/wp-calypso/tree/master/server/pragma-checker) will complain if this code is ever used server side.